lldb/source/Symbol/SymbolContext.cpp

```diff
@@ -89,13 +97,18 @@ bool SymbolContext::DumpStopContext(Stream *s, ExecutionContextScope *exe_scope,
       s->Printf("<");
       dumped_something = true;
     } else {
-      ConstString name;
+      ConstString name_func;
```

Comment:
This is what I mean about the name clashes too.

You're calling the function's name name and the pattern used to match "name". It's not a name though, it's a pattern. So if you can get to a stable place with the rest of the changes, I highly recommend changing `name` to `pattern` throughout.

Reply:
Also name_func sounds like a function you call to get a name not the name of a function which would be func_name if anything.

But I digress, you see my point.

Reply:
Since the argument is now named `pattern`, I think this confusion is cleared.